compiler/rustc_feature/src/active.rs

```diff
@@ -126,7 +126,7 @@ declare_features! (
     /// macros disappear).
     (active, allow_internal_unsafe, "1.0.0", None, None),
 
-    /// no-tracking-issue-end
+    // no-tracking-issue-end
```

Comment:
Why did this change?

Reply:
It was a bug in the original code. Currently [the docs](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_feature/struct.Features.html#structfield.link_llvm_intrinsics) look like:

![image](https://user-images.githubusercontent.com/37223377/107907018-6f700a80-6f07-11eb-8aed-4ddefcc99b6d.png)

because the doc comment is processed as being attached to `link_llvm_intrinsics`.